multi-arch-builders/coreos-ppc64le-builder.bu

```diff
@@ -23,3 +23,7 @@ storage:
       overwrite: true
       contents:
         inline: coreos-ppc64le-builder
+systemd:
+  units:
+    - name: zincati.service
```

Comment:
Can we add a comment linking to https://github.com/coreos/fedora-coreos-tracker/issues/1633 here? And similarly below.